frontend/ts/submission.ts

```diff
@@ -22,19 +29,17 @@ hs.initHighlightingOnLoad();
               penalty: number;
           };
 
-    async function fetchResult(id: string): Promise<Result> {
-        return (
-            await fetch(`/admin/submissions/${id}/verdict`)
-        ).json() as Promise<Result>;
-    }
+    const fetchResult = async (id: string): Promise<Result> => {
```

Comment:
I'm not really sure why this is better?

Reply:
@natsukagami this avoids duplicating return types. Also it seems that a large part of the codebase uses arrow function expressions anyways, so unifying the style would be better.